### PR TITLE
Fix/sm 63 mapper item except

### PR DESCRIPTION
**Pull request by ievgenvarava**
Moved itemExcept logic from mapper configuration into dedicated translator functions

### CODE REVIEW

File: src/SprykerMiddleware/Zed/Process/Business/Translator/TranslatorFunction/ExcludeKeysAssociativeFilter.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+use SprykerMiddleware\Zed\Process\Business\Exception\WrongTypeValueTranslatorException;
+
+class ExcludeKeysAssociativeFilter extends AbstractTranslatorFunction implements TranslatorFunctionInterface
+{
+    /**
+     * @var array
+     */
+    protected $requiredOptions = [
+        'excludeKeys',
```
Comment (valerio8787) [inline src/SprykerMiddleware/Zed/Process/Business/Translator/TranslatorFunction/ExcludeKeysAssociativeFilter.php]: Please use constant for option name

**Review by valerio8787 — COMMENTED** [review]
Please fix my comment

**Review by valerio8787 — APPROVED** [review]

**Review by vlunov-spryker — APPROVED** [review]

File: src/SprykerMiddleware/Zed/Process/Business/Mapper/Mapper.php [hunk 2]
```diff
@@ -124,21 +121,7 @@ protected function mapCallable(array $result, array $payload, string $key, calla
     protected function mapArray(array $result, array $payload, string $key, array $value): array
```
Comment (valerio8787) [inline src/SprykerMiddleware/Zed/Process/Business/Mapper/Mapper.php]: Seems that mapKey and mapArray methods are similar now. May we need to leave only one if there aren't any difference

Comment (ievgenvarava) [reply src/SprykerMiddleware/Zed/Process/Business/Mapper/Mapper.php]: fixed

**Review by valerio8787 — APPROVED** [review]

**Pull request merged**